integration/sql/iface-java/src/test/java/io/openlineage/sql/OpenLineageSqlTest.java

```diff
@@ -39,7 +39,9 @@ void parseWithDialect() {
     assertEquals(
         output,
         new SqlMeta(
-            Arrays.asList(new DbTableMeta("random-project", "dbt_test1", "source_table")),
+            Arrays.asList(
+                new DbTableMeta(
+                    "random-project", "dbt_test1", "source_table", new Quotes("`", "`", "`"))),
```

Comment:
Why can't we follow the convention from `sqlparse-rs` and have quoting information included within the identifier?
In this case, in `DbTableMeta` each of the properties `schema`, `database` or `name` would be of `Ident` type instead of string, and each ident will contain information whether quote is used or not. 

I don't understand why `DbTableMeta` has all three properties repeated within `Quotes` class.

Reply:
We need both Java and Python interfaces to be backwards compatible. For that reason we can't replace `database`, `schema` and `name` properties from string to structure (which `Ident` really is).

We could have another constructor for `DbTableMeta` that takes `Ident`s as arguments. I can't see real example of someone using it though.

Reply:
Now I get it. Thanks!